scripts/propose.py

```diff
@@ -1,18 +1,16 @@
 import grpc
 import logging
-import sys
 from rchain.client import RClient
 from rchain.client import RClientException
 
 root = logging.getLogger()
 root.setLevel(logging.DEBUG)
-error=0
+
 with grpc.insecure_channel('localhost:40401') as channel:
     client = RClient(channel)
     try:
         client.propose()
     except RClientException as e:
         # NoNewDeploys is a valid scenario
         if "NoNewDeploys" not in str(e):
-            error=1
-sys.exit(error)
\ No newline at end of file
+            raise
```

Comment:
I guess we don't want to any type of Exception to be shown - as we won't handle it any way?